### DISCUSSION TITLE
Support for multiple account numbers?

### DISCUSSION THREAD

**Discussion by erictpeterson** [homebridge-simplisafe3/homebridge-simplisafe3#441] [Feature Requests]
I have two SimpliSafe systems under a single account. The plug-in can see both of them but the second isn't reporting back correctly (e.g., in Apple Home it just reports "disarming" constantly). Is there a way to support both systems? Or can I install the plug-in twice, once for each account number?

**Comment by shamoon** [comment]
I think if you use child bridges you can install it twice, but Im not 100% certain about that
